src/panels/lovelace/cards/hui-picture-card.ts

```diff
@@ -86,10 +87,14 @@ export class HuiPictureCard extends LitElement implements LovelaceCard {
           hasHold: hasAction(this._config!.hold_action),
           hasDoubleClick: hasAction(this._config!.double_tap_action),
         })}
-        tabindex="0"
+        tabindex=${ifDefined(hasAction(this._config.tap_action))
```

Comment:
tabindex=${ifDefined(hasAction(this._config.tap_action)
          ? "0"
          : undefined)}